src/utils/pgwire/src/pg_extended.rs

```diff
@@ -238,6 +238,10 @@ pub struct PreparedStatement {
     param_types: Vec<TypeOid>,
 }
 
+lazy_static::lazy_static!{
```

Comment:
> Though it's not the bottleneck for this case, I think we can use a pre-compiled regex with LazyLock here.

Seems once_cell is an unstable feature. To prevent to use unstable features as much as possible, I think using lazy_static here is acceptable 🤔.  @BugenZhao

Reply:
Don't worry about that. We are relying on unstable features in tons of places. 🤣

- #5385